lib/lumberjack/beats/server.rb

```diff
@@ -485,6 +485,14 @@ def read_socket(&block)
       end
     end
 
+    def normalize_v1_metadata_encoding(map)
+      # lets normalize the metadata of the v1 frame to make
+      # sure everything is utf-8 minus the content itself.
+      # We need this to make this plugin backward compatible.
```

Comment:
LSF is a weird beast, it doesn't return the metadata content in utf-8 (path.. offset) etc and this make logstash crash.
If you look at the lumberjack input https://github.com/logstash-plugins/logstash-input-lumberjack/blob/master/lib/logstash/inputs/lumberjack.rb#L112, I though it was best to move it closer to the actual data parsing. At that stage we know its from LSF or filebeat.
On the other hand filebeat is good citizen :(